AndroidBound/src/main/java/solutions/alterego/androidbound/android/BoundActivityDelegate.java

```diff
@@ -137,40 +150,35 @@ public void onCreate(Bundle savedInstanceState) {
             mShouldCallCreate = true;
             mCreateBundle = savedInstanceState;
         }
-    }
 
-    @Override
-    public void onStart() {
-        if (getViewModels() != null) {
-            for (ViewModel viewModel : getViewModels().values()) {
-                viewModel.onStart();
-            }
+        if (getBoundActivity() != null && getBoundActivity().getIntent() != null) {
+            onNewIntent(getBoundActivity().getIntent()); //we call this manually so that you don't have to check in ViewModel.onCreate()
         }
     }
 
     @Override
-    public void onRestart() {
-        if (getViewModels() != null) {
-            for (ViewModel viewModel : getViewModels().values()) {
-                viewModel.onRestart();
-            }
-        }
+    public boolean isCreated() {
+        return true;
```

Comment:
because it's a part of the `IActivityLifecycle` but that part was meant more for ViewModels i.e. to signal if they had the `onCreate()` called yet or not. here the creation is done in the constructor so I always consider them created.